src/ast/mod.rs

```diff
@@ -2759,6 +2759,15 @@ pub enum Statement {
         version: Option<Ident>,
     },
     /// ```sql
+    /// DROP EXTENSION [ IF EXISTS ] name [, ...] [ CASCADE | RESTRICT ]
+    /// ```
```

Comment:
Could we also include the link in the description here? would help folks find the syntax quicker in the future if needed

Reply:
Done!